packages/vscode-extension/project.json

```diff
@@ -46,7 +46,7 @@
 					"cp ./LICENSE dist/packages/vscode-extension",
 					"cp packages/vscode-extension/package.json dist/packages/vscode-extension",
 					"cp packages/vscode-extension/README.md dist/packages/vscode-extension",
-					"cp node_modules/@php-wasm/node/*.wasm dist/packages/vscode-extension"
+					"cp -r node_modules/@php-wasm/node/*_*_* dist/packages/vscode-extension"
```

Comment:
7_* and 8* didn't work after all? I think this will also bring over the `php_8_0.js` etc files which isn't a big deal but maybe there's an easy way not to do that

Reply:
It worked, I was over optimizing things  😅 .

Changed to `7_*` and `8_*` https://github.com/WordPress/playground-tools/pull/141/commits/305f477a5dbbc2cf32936f04eeca5e188b901898